episodes/2015-12-16/index.js

```diff
@@ -17,7 +17,8 @@ export default {
       ],
       picks: [
         `[Javascript Allonge](https://leanpub.com/javascript-allonge)`,
-        `[Hello Declarative World!](http://codon.com/hello-declarative-world)[my ES6 implementation](https://github.com/ashleygwilliams/relational-node)`,
+        `[Hello Declarative World!](http://codon.com/hello-declarative-world)`,
```

Comment:
Actually, I think these are related and should be on one line. Just needs to have a separator. Maybe ` - `?